src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue

```diff
@@ -29,7 +29,7 @@
 		<template #icon>
 			<Plus :size="20" />
 		</template>
-		{{ $t('calendar', 'New event') }}
+		{{ $t('calendar', 'Event') }}
```

Comment:
The button should have a title or aria-label with a more descriptive text than "Event"

Reply:
Understood @nickvergessen. 
Please check my additional commit. 
It's the first time I managed that. 
Is it correctly done and enough ?